workflow-controller/controller-service/src/main/java/io/hyscale/controller/model/WorkflowContext.java

```diff
@@ -32,6 +32,7 @@
     private String appName;
     private String serviceName;
     private String envName;
+    private String kubeConfigPath;
```

Comment:
The type should be AuthConfig right to cover all cases

Reply:
@SameeraGrandhi Your views on this?

Reply:
Yes , it should be authConfig